packages/react-aria-components/docs/ComboBox.mdx

```diff
@@ -1474,3 +1475,38 @@ function ComboBoxClearButton() {
 ### Hooks
 
 If you need to customize things even further, such as accessing internal state, intercepting events, or customizing the DOM structure, you can drop down to the lower level Hook-based API. See [useComboBox](useComboBox.html) for more details.
+
+## Testing
+
+### Test utils <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
```

Comment:
Do we need this sub-heading if that's the only one? Maybe just put the version badge next to "Testing"?

Reply:
I mainly did this for parity with the [RSP component testing pages](https://reactspectrum.blob.core.windows.net/reactspectrum/64e1b71410c757c6caaa4d978bbd3228602b7138/docs/react-spectrum/ComboBox.html#testing) that had non test util content separated from the test util content via the subheading, but I'm fine removing the subheading in general

Reply:
actually, on 2nd thought, there are [components](https://reactspectrum.blob.core.windows.net/reactspectrum/64e1b71410c757c6caaa4d978bbd3228602b7138/docs/react-spectrum/ColorArea.html#testing) that have the "Testing" session but don't yet have test utils. Feels weird to omit the version badge but then add it when they eventually get test utils so I'm leaning towards keeping the sub header to make it obvious that the test utils themselves are in alpha